web/src/app/admin/admin-outgoing-logs/OutgoingLogsList.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { useEffect } from 'react'
+import { Box } from '@mui/system'
+import { DateTime } from 'luxon'
+import { DebugMessage } from '../../../schema'
+import OutgoingLogCard from './OutgoingLogCard'
+import { useFuse } from './hooks'
+import { useURLParam } from '../../actions'
+import { Typography, Button } from '@mui/material'
+
+export const LOAD_AMOUNT = 50
+
+interface Props {
+  debugMessages?: DebugMessage[]
+  selectedLog: DebugMessage | null
+  onSelect: (debugMessage: DebugMessage) => void
+}
+
+export default function OutgoingLogsList(props: Props): JSX.Element {
+  const { debugMessages = [], selectedLog, onSelect } = props
+
+  const [searchTerm] = useURLParam('search', '')
+  const [start] = useURLParam('start', '')
+  const [end] = useURLParam('end', '')
+  const [limit, setLimit] = useURLParam<string>('limit', '1')
+
+  const { setSearch, results } = useFuse<DebugMessage>({
+    data: debugMessages,
+    keys: ['destination', 'userName', 'serviceName', 'status'],
+    options: {
+      shouldSort: false,
+      showResultsWhenNoSearchTerm: true,
+      ignoreLocation: true,
+    },
+  })
+
+  let filteredResults = results.slice() // copy results array
+  filteredResults = filteredResults.filter((result) => {
+    if (!start && !end) return true
+
+    const startDT = DateTime.fromISO(start)
+    let endDT = DateTime.fromISO(end)
+    const createdAtDT = DateTime.fromISO(result.item.createdAt)
+
+    if (start && !end) {
+      endDT = DateTime.now()
```

Comment:
We should update the logic to ignore end time if it's not set instead of using `DateTime.now()` -- this is going to potentially filter items based on the user's clock & clock drift when the input field is blank. There should be no filtering on end time if the value is empty/null.